packages/portia/portia.1.1/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "1.2"
+name: "portia"
+version: "1.1"
+maintainer: "Cedric Cellier <rixed@happyleptic.org>"
+authors: "Cedric Cellier <rixed@happyleptic.org>"
+homepage: "https://github.com/rixed/portia"
+doc: "http://rixed.github.io/portia/"
+dev-repo: "https://github.com/rixed/portia.git"
+bug-reports: "mailto:rixed-opam@happyleptic.org"
+build: [[make]]
+build-doc: [[make "doc"]]
+build-test: [[make "check"]]
+install: [make "install"]
+remove: ["ocamlfind" "remove" "portia"]
+depends: [
+  "ocamlfind"
```

Comment:
Could you add the ```{build}``` attribute to ocamlfind ? This would avoid rebuilding this package and its dependencies in case ocamlfind is to be upgraded.

Reply:
Sure, thank you for the tip.
I updated the commit.